src/Pages/Header/Header.tsx

```diff
@@ -1,25 +1,27 @@
 import './Header.scss';
 import { Link } from 'react-router-dom';
+import { RootState } from '../../modules/redux/store';
+import { useSelector } from 'react-redux';
 import Dark from './Dark.svg';
 import Logo from './Digiency.svg';
 import MenuIcon from './menuIcon.png';
 
 export const Header = () => {
+  const login = useSelector((state: RootState) => state.login);
+
+  // console.log(login.isSigned);
+
   return (
     <section className="header">
       <div className="container">
         <img className="logo" src={Logo} alt="" />
         <div className="navbar">
-          <Link to="/">
-            <a href="/#">Home</a>
-          </Link>
+          <Link to="/">Home</Link>
           <a href="/#">About Us</a>
           <a href="/#">Our Work</a>
           <a href="/#">Clients</a>
-          <Link to="/our-blog">
-            <a href="/#">Our Blog</a>
-          </Link>
-          <a href="/#">Profile</a>
+          <Link to="/our-blog">Our Blog</Link>
+          <Link to="/login">{login.isSigned ? 'Bextiyar' : 'Profile'}</Link>
```

Comment:
I think it should be like this:
{login.isSigned ? <Link to="/profile">Profile</Link> : <Link to="/login">Log in</Link>}

Reply:
fixed